llpc/include/llpc.h

```diff
@@ -109,14 +109,24 @@ struct ShaderModuleBuildOut {
   ShaderModuleData *pModuleData; ///< Output shader module data (opaque)
 };
 
+enum StageCacheInfo : uint8_t {
+  CacheNotChecked = 0, ///< Stage cache is not checked.
+  CacheMiss,           ///< Stage cache miss.
+  CacheHit,            ///< Stage cache hit.
+};
+
 /// Represents output of building a graphics pipeline.
 struct GraphicsPipelineBuildOut {
   BinaryData pipelineBin; ///< Output pipeline binary data
+  bool pipelineCacheHit;  ///< Indicates whether pipeline cache access was a hit
```

Comment:
You use 3 values to describe stage cache accesses (`ShaderCacheInfo`) and only 2 values to describe pipeline cache access (hit/miss). Does the code always check pipeline cache, or does the vulkan API force us to return true/false here?